tests/conftest.py

```diff
@@ -283,13 +283,12 @@ def pytest_generate_tests(metafunc: pytest.Metafunc) -> None:
         return
 
     import re
-    from copy import deepcopy
     from pathlib import Path
 
     import yaml
 
     def get_version_name(uri: str) -> tuple[str, str]:
-        """Return the version and name part of a uri."""
+        """Return the version and name part of a TEAM4.0-style URL."""
```

Comment:
"TEAM4.0-style URL" -- is that a thing? :question:

Reply:
I meant the <namespace>/<version>/<name> and URI's always being resolvable URLs as well, usually (but not always) starting with `http://onto-ns.com/meta/`.